data/10.1101/2022.11.10.516056/2022.11.10.516056.xml

```diff
@@ -370,8 +370,7 @@
 <ref id="c106"><mixed-citation publication-type="other"><string-name><surname>Zeiler</surname> <given-names>SR</given-names></string-name>, <string-name><surname>Hubbard</surname> <given-names>R</given-names></string-name>, <string-name><surname>Gibson</surname> <given-names>EM</given-names></string-name>, <string-name><surname>Zheng</surname> <given-names>T</given-names></string-name>, <string-name><surname>Ng</surname> <given-names>K</given-names></string-name>, <string-name><surname>O&#x2019;Brien</surname> <given-names>R</given-names></string-name>, <string-name><surname>Krakauer</surname> <given-names>JW</given-names></string-name>. <article-title>Paradoxical Motor Recovery From a First Stroke After Induction of a Second Stroke: Reopening a Postischemic Sensitive Period</article-title>. <source>Neurorehabil Neural Repair</source>. <year>2015</year> 12; doi: <pub-id pub-id-type="doi">10.1177/1545968315624783</pub-id>.</mixed-citation></ref>
 <ref id="c107"><mixed-citation publication-type="journal"><string-name><surname>Zhang</surname> <given-names>T</given-names></string-name>, <string-name><surname>Pan</surname> <given-names>N</given-names></string-name>, <string-name><surname>Wang</surname> <given-names>Y</given-names></string-name>, <string-name><surname>Liu</surname> <given-names>C</given-names></string-name>, <string-name><surname>Hu</surname> <given-names>S.</given-names></string-name> <article-title>Transcranial Focused Ultrasound Neuromodulation: A Review of the Excitatory and Inhibitory Effects on Brain Activity in Human and Animals</article-title>. <source>Frontiers in Human Neuroscience</source>. <year>2021</year> 9; <volume>15</volume>. doi: <pub-id pub-id-type="doi">10.3389/fnhum.2021.749162</pub-id>.</mixed-citation></ref>
 </ref-list>
-<app-group>
-<app id="app1">
+<sec id="app1">

```

Comment:
appendices are not handled at all yet in encoda. This would at least expose them to client until encoda are able to work on this. Would be useful to link to an issue raised with Encoda.